sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -207,7 +208,27 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
           existingOffering.exceptionalExpenses;
 
         offering.offeringType = OfferingTypes.ScholasticStanding;
-
+        if (offering.studyBreaks?.studyBreaks) {
+          // Study Breaks calculation.
+          const adjustedStudyBreak =
+            EducationProgramOfferingService.adjustStudyBreaks(
```

Comment:
I know this ticket's scope is only about the study breaks. 

What about the the other validations with respect to offering end date? e.g. is the new offering end date validated to be after the offering start date? (I submitted an application with new end date before offering start date and it created an offering) what about the offering minimum days validation ? is it expected to be bypassed in scholastic standing scenario?